src/Microsoft.Health.Dicom.Web/Startup.cs

```diff
@@ -52,17 +55,20 @@ public virtual void Configure(IApplicationBuilder app)
     }
 
     /// <summary>
-    /// Adds ApplicationInsights for telemetry and logging.
+    /// Adds Open telemetry exporter for Azure monitor.
     /// </summary>
-    private void AddApplicationInsightsTelemetry(IServiceCollection services)
+    private void AddOpenTelemetry(IServiceCollection services)
```

Comment:
Let's change the configuration as well from `ApplicationInsights` to `AzureMonitor`.

Reply:
Discussed this offline and we dont have to change this configuration since we are using ApplicationInsights dashboard only.